plugins/ui/noice.nix

```
@@ -132,34 +180,30 @@ with lib;
         enabled = helpers.defaultNullOpts.mkBool true "enable LSP progress";
 
         format =
-          helpers.defaultNullOpts.mkNullable (types.either types.str types.anything) ''"lsp_progress"''
+          helpers.defaultNullOpts.mkNullable (types.either types.str types.anything) "lsp_progress"
             ''
               Lsp Progress is formatted using the builtins for lsp_progress
             '';
         formatDone =
-          helpers.defaultNullOpts.mkNullable (types.either types.str types.anything) ''"lsp_progress"''
+          helpers.defaultNullOpts.mkNullable (types.either types.str types.anything) "lsp_progress"
             "";
 
-        throttle = helpers.defaultNullOpts.mkNum "1000 / 30" "frequency to update lsp progress message";
+        throttle = helpers.defaultNullOpts.mkNum (1000 / 30) "frequency to update lsp progress message";
```

```suggestion
        throttle = helpers.defaultNullOpts.mkNum (1000 / 30) "frequency to update lsp progress message";
```
Wouldn't this appear as the result of 1000 / 30 (i.e. 100) ?

---

Yes. Without adding "pluginDefaultText", or some other way to pass pre-formatted text, I couldn't see an alternative.

---

Maybe we could add a special case in `mkDesc` for `literExpression` types.

That might be easier to use with our existing helpers rather than forcing the use of `defaultText` 🤔